### ISSUE TITLE
dragon reset does not reset anim

### ISSUE THREAD

**Issue by 20scrolls** [us3rT0m/OnlyUP-Trainer#7]
I noticed that the reset drake button does not reset the dragons animation, which makes practicing the skips much harder. Any way to implement this?

**Comment by wRadion** [comment]
Yes, we don't know yet how to do that, otherwise it would have been the thing to do trust me x).

The easiest thing to do I guess is to find a timer value or something that's controlling where you are on the animation and reset this value to 0. But we don't know where it is, and we don't even know if this timer exists.
